### PR TITLE
Fix render README action

### LINKED ISSUE #443: Fix `render-readme.yml` action

### PR DESCRIPTION
**Pull request by jdhoffa**
Closes #443

### CODE REVIEW

**Comment by jdhoffa** [comment]
Relates to https://github.com/RMI-PACTA/Actions/issues/11

**Comment by jdhoffa** [comment]
Seems like `r-lib/actions/setup-renv@v2` is failing

File: .github/workflows/render-readme.yaml [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
-          install.packages("r2dii.data")
-          install.packages("remotes")
-          remotes::install_local(".")
-          remotes::install_cran("rmarkdown")
-        shell: Rscript {0}
-      - name: Render README
-        run: Rscript -e 'rmarkdown::render("README.Rmd")'
-      - name: Commit results
+      - name: Checkout repo
+        uses: actions/checkout@v3
```
Comment (cjyetman) [inline .github/workflows/render-readme.yaml:16]:
```suggestion
        uses: actions/checkout@v4
```
https://github.com/actions/checkout/releases

Comment (jdhoffa) [reply .github/workflows/render-readme.yaml]: Thank you
**shakes fist vaguely at r-lib**

**Comment by jdhoffa** [comment]
That seems to do it! https://github.com/RMI-PACTA/r2dii.match/actions/runs/7713272963

File: .github/workflows/render-readme.yaml [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
 
-name: Render README
+name: render-readme
 
 jobs:
-  render:
-    name: Render README
-    runs-on: macOS-latest
+  render-readme:
+    runs-on: ubtu-latest
```
Comment (cjyetman) [inline .github/workflows/render-readme.yaml:11]:
```suggestion
    runs-on: ubuntu-latest
```

**Comment by jdhoffa** [comment]
Sorry for the spam @cjyetman , converted back to draft. This is driving me nuts lol

**Comment by jdhoffa** [comment]
> Sorry for the spam @cjyetman , converted back to draft. This is driving me nuts lol

Maybe silly to put so much work into it, but I'm committed now XD

**Comment by jdhoffa** [comment]
Ok it works now lol:
https://github.com/RMI-PACTA/r2dii.match/actions/runs/7715397603/job/21029870364

and only takes 2min

**Review by cjyetman — APPROVED** [review]

**Pull request merged**